### ISSUE TITLE
Organize Computable Features in a Reference Catalogue

### ISSUE THREAD

**Issue by LinguList** [cldf/cltoolkit#30]
The basic idea would be to make a collection of features we can compute (from text, from igt, from wordlists, etc.). This could inform some Grammaticon at some point in time, but for now, we would concentrate in collecting features which can be computed, provide links to functions that compute them, along with the data one needs.

In some sense, cltoolkit already has some structure for this:

https://github.com/cldf/cltoolkit/blob/10c2692bd5370d042fc70494a0b948fea8e6a58d/src/cltoolkit/features/__init__.py#L18-L40

On the long run, however, this feature collection could have a live of its own and include also other implementations of features/language structures, e.g., also in pyigt, and the like.

@fmatter, we discussed this quickly with @xrotwang, do you think it would be interesting to follow this up? The idea would be to say something like: language structure X which is the same as WALS AX and APICS N can be computed from a wordlist with the function XYZ and from IGT with the function ABC. It would be a kind of Concepticon with very limited scope.

We still discuss the name, we had something like "struca" or "cls" (Catalog of Language Structures), "calas" may also be an option. 

Structurewise, it would be a list of feature IDs that link to datasets like WALS, APICS, and also to code implementations.

**Comment by fmatter** [comment]
Only getting around to this now... 

> The idea would be to say something like: language structure X which is the same as WALS AX and APICS N can be computed from a wordlist with the function XYZ and from IGT with the function ABC. It would be a kind of Concepticon with very limited scope.

So it would be like a recipe for establishing certain typological features, using data available as CLDF and functions from various corners of the CLDF ecosystem?

**Comment by LinguList** [comment]
Yep. All what your phantasy allows. We tested phoneme inventories so far, but also cases like "mama vs. Papa" in the world's languages, etc. All what one may think of, but we'd be clear regarding the data needed. For mama vs. papa you need lexemes, for SVO you need IGT or a treebank (?), I suppose, etc.
